src/app/builder/before_mount.rs

```diff
@@ -42,13 +42,13 @@ impl MountPoint for web_sys::HtmlElement {
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
 pub enum MountType {
     /// Take control of previously existing elements in the mount. This does not make guarantees of
-    /// elements added after the [`App`] has been mounted.
+    /// elements added after the [`crate::app::App`] has been mounted.
```

Comment:
I want to revisit all docs once a newer better way of writing links is available on stable Rust - don't spend too much time with improving documentation links.